clients/fides-js/src/lib/i18n/i18n-utils.ts

```diff
@@ -338,7 +338,7 @@ export function setupI18n(): I18n {
     },
 
     load: (locale: Locale, messages: Messages): void => {
-      allMessages[locale] = { ...messages, ...allMessages[locale] };
+      allMessages[locale] = { ...allMessages[locale], ...messages };
```

Comment:
I will *never* be able to remember the precedence of the object spread operator.